### ISSUE TITLE
User story labels do not define officers with enough granularity

### ISSUE THREAD

**Issue by GreenCChampion** [folklabs/connecting-communities#6]
I think that we need to be quite specific re our early stakeholders, and not assume all officers have the same needs. 
I would like to add 
1. a comms person - discover major stakeholders on a particular subject eg housing to increase influence on the subject
2. an active communities officer - network key contacts in a particular area so they can increase forum participation
3. a regeneration officer/consultant - identify key organisations in a particular area impacted by regeneration so they can coproduce solutions together
4. a Streetcare officer - access key contacts in a particular area - add new contacts to database but restrict who else can contact them

**Comment by djwesto** [comment]
Thanks for adding this @GreenCChampion (and assigning the issue to me – I can see you're clearly getting to grips with using GitHub). This is incredibly helpful. I will try to add them in as new user stories but will probably need a lot more detail from you to open out what each of these stories means.

Can we assume these four officers you've added are in order of priority for Phase 1?

**Comment by djwesto** [comment]
I want to unpack each of the four new use cases you've requested in this issue. Right now they're not clear enough for me to actually turn them into user stories we can try to deliver for you.

Let's take them in order. Number 1 seems to basically be saying the following:

> As a **Communications Officer**, I want to search or browse for records by category so that I can ensure I reach and influence the maximum number of stakeholders

This is already one of the features we're exploring in Phase 1 – the ability to browse and search for local organisations by category or categories assigned to them (e.g. _Parks & Gardens_, _Health & Wellbeing_, _Markets_).

Once the Communications Officer has conducted a search, she can download a CSV file to be imported into Outlook for a mass email.

Does this sound right?

**Comment by djwesto** [comment]
Next, here's your second use case expressed as a user story:

> As an **Active Communities Officer**, I want to network key contacts in a particular area so that they can increase forum participation

This one is not as clear to me as the first. Could you possibly add more detail on what you mean by "network"? To me, it seems as if you're hoping that the Active Communities Officer would be able to forward the details of each "key contact" on to all the other key contacts? Is that what you mean by "networking" them together?

If this is wrong, could you possibly reframe it in terms of what you'd like to see happen as an ideal outcome of this user story?

**Comment by djwesto** [comment]
Use case three says effectively:

> As a **Regeneration Officer**, I want to browse or search for records organisations in a particular geographical area

This seems to be fairly similar to the first story except the Regeneration Officer needs to be able to search or browse a geographical area. Correct?

We'd need more information than this on how they'd need to perform their search. There are lots of ways of searching by geography – some simple and others are very complex, as you may recall from COD.
- Would it be enough to search or browse records for identical or similar post codes?
- Would they simply look on a map?
- Would they be looking for the address of the organisation or the catchment area they work on?
- Would it be enough to simply search for all records in a town, ward or street?

**Comment by djwesto** [comment]
Finally:

> As a **Streetcare Officer**, I want to add new records and restrict who else can edit them so that...? _(is this about protecting each officer's records?)_

This option again needs a bit more detail before we can see what you're really asking for. Could you explain the sort of issues which would arise from having unrestricted access to all the records? This will help us understand how you'd want access to be restricted.
- Are you worried that some officers might accidentally make changes or delete records by mistake?
- Are you worried that there would be no way to see where changes had been made?
- Are some officers protective of their contacts and reluctant to allow others to change them?
- Can we assume that there would be a restricted number of officers with global access?
- Why is this particular to Streetcare Officers specifically, as opposed to other officers? Presumably other officers would also benefit from having their records protected?
- What happens when two officers own the same record? Can they both edit or does one override the other?

**Comment by djwesto** [comment]
Sorry for all the questions @GreenCChampion – I hope you can see there's a lot to think about.

My overall comment is that I don't necessarily think that this set of user stories justifies us adding a new label onto this Github project – one for each officer. We tend to try to keep labels as general as possible so our **as-officer** label can arguably cover all officers – the labels are really just to help us sort and filter out individual tasks and issues – they're not part of the project specification.

However, when we do write up each of these user stories, I would _absolutely_ want to ensure we clarify which officer we're talking about and your answers here will be immensely helpful.

**Comment by GreenCChampion** [comment]
The other priority officer is the Commissioner. 
Re Streetcare officer - I need to test all this out with the officer in mind. I think he is recruiting individuals who won't want to be bombarded by emails. This is going to be a recurring theme..
sue

**Comment by djwesto** [comment]
If it's a recurring theme then it needs to be captured in an issue. I'll go set one up now.

**Comment by GreenCChampion** [comment]
Re the regeneration officer - often these are consultants so if they could draw an geographic area on a map and then select all entries within that area and download them to a csv file that would be good. 
It would also be good to identify who else is talking to people in that area - how can we do that? Can we alert people when we see that similar downloads have been made by someone else?

**Comment by GreenCChampion** [comment]
Another user could be the local GP or their patient participation group. We could have a real life example to work with very soon. They want to be able to direct people to local 'wellbeing' places and activities eg community gardens, reading group, walking group.

**Comment by GreenCChampion** [comment]
I have created separate discussions around each user story to allow us to take this work forward
